example/resources/tags.js

```diff
@@ -4,7 +4,7 @@ jsonApi.define({
   namespace: "json:api",
   resource: "tags",
   description: "Used to group resources together, useful for finding related resources.",
-  handlers: jsonApi.mockHandlers,
+//  handlers: jsonApi.mockHandlers,
```

Comment:
Well it makes it harder for the test of changing the default handler.  That's why a "commented out" the code.  But I do see your point.  Will try to update the patch sometime tomorrow my time (I'm in NZ +13:00 UTC).